second-analysis-steps/rerun-stripping.md

```diff
@@ -56,9 +56,15 @@ sc = StrippingConf(Streams=[custom_stream],
                    BadEventSelection=filterBadEvents)
 ```
 
+ - Updating the path for `dtt.Inputs`: the output is now placed directly in `Phys`
+ ```python
+ dtt.Inputs = ['/Event/Phys/{0}/Particles'.format(line)]
+ ```
+
  - Inserting the node killer and the stripping selection sequence into the Gaudi sequence
 
 ```python
 DaVinci().appendToMainSequence([event_node_killer, sc.sequence()])
 ```
 
+It is strongly recommended to run restripping with the same version of DaVinci as the one used for the corresponding stripping production: this ensures all the settings are configured the same exact way as for your data. For the Stripping 28r1, DaVinci v41r4p4 was used. Notice that this version is not available for the default platform `x86_64-slc6-gcc62-opt`, and you have to change the platform before launching DaVinci: `LbLogin -c x86_64-slc6-gcc49-opt`.
```

Comment:
What do you think not recommending `LbLogin` and instead saying:

> Notice that this version is not available for the default platform `x86_64-slc6-gcc62-opt`. To work around this you can specify the platform you want to use using `lb-run -c x86_64-slc6-gcc49-opt DaVinci/v41r4p4 ...` or by [setting the `application.platform` attribute in ganga](https://lhcb.github.io/starterkit-lessons/first-analysis-steps/davinci-grid.html#picking-up-a-right-platform).

We could even suggest using `-c best` with `lb-run`.

Reply:
Well spotted, thanks! 
I like the `-c best` option.